trailblazer/constants.py

```diff
@@ -9,4 +12,51 @@
 STATUS_OPTIONS = ("pending", "running", "completed", "failed", "error", "canceled", "completing")
 JOB_STATUS_OPTIONS = SLURM_NORMAL_CATEGORIES + SLURM_FAILED_CATEGORIES
 PRIORITY_OPTIONS = ("low", "normal", "high", "express", "maintenance")
-TYPES = ("wes", "wgs", "rna", "tgs", "other")
+TYPES = ("other", "rna", "tgs", "wes", "wgs", "wts")
+
+
+class TaskManager(Enum):
+    """Supported task managers."""
+
+    SLURM: str = "Slurm"
+    TOWER: str = "NF_Tower"
+
+    @classmethod
+    def list(cls):
+        return [task.value for task in cls]
+
+
+class TrailblazerStatus(Enum):
+    """Trailblazer allowed status."""
+
+    PENDING: str = "pending"
+    RUNNING: str = "running"
+    COMPLETED: str = "completed"
+    FAILED: str = "failed"
+    ERROR: str = "error"
+    CANCELLED: str = "cancelled"
+    COMPLETING: str = "completing"
+
+
+TOWER_STATUS: Dict[str, str] = {
+    "SUBMITTED": TrailblazerStatus.PENDING.value,
```

Comment:
They are always constant values that's why I the upper case. I think a dict is better in that case as I can iterate over and use the `get` method. Enum could have been good if it would allowed for duplicate values, but since it doesn't then I can't use it. I've seen the is a `aenum` module which would allow this, but would need to be added to requirements, so maybe I dict is enough?

Reply:
👍